pkg/start/start.go

```diff
@@ -194,7 +196,29 @@ func Start(ctx context.Context, inst *store.Instance) error {
 
 	begin := time.Now() // used for logrus propagation
 
-	if err := haCmd.Start(); err != nil {
+	if launchHostAgentForeground {
+		logrus.Info("Running the host agent in the foreground")
+		if isatty.IsTerminal(os.Stdin.Fd()) || isatty.IsCygwinTerminal(os.Stdin.Fd()) {
```

Comment:
I wonder if it is time to create a wrapper for this condition:

```console
$ ag IsCygwin
cmd/limactl/main.go
82:		if runtime.GOOS == "windows" && isatty.IsCygwinTerminal(os.Stdout.Fd()) {

cmd/limactl/list.go
171:		if isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd()) {

cmd/limactl/shell.go
176:	if isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd()) {

pkg/progressbar/progressbar.go
15:	if isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd()) {

pkg/qemu/entitlementutil/entitlementutil.go
88:		if isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd()) {
```

Not sure what the right package is, maybe `uiutil.isatty()`?

Reply:
Yes, but it can be another PR